### ISSUE TITLE
Help installing the appropriate stub version.

### ISSUE THREAD

**Issue by DMunkei** [typeddjango/djangorestframework-stubs#662]
Hey, I'm not understanding which version of the stubs I need to install?

These are my installed modules per `pip list`.

```bash
django-stubs              4.2.7
django-stubs-ext          5.0.4
djangorestframework       3.14.0
Django                    4.2.9
mypy                      1.7.1
mypy-extensions           1.0.0
```
If I'm understanding the docs right. then I am supposed to install the `1.7.0` version to match the `mypy` version I have?

Would the command look like this?

```code
pip install djangorestframework-stubs==1.7.0
```
When I open my editor `NeoVim` with the virtual environment active I get this error message for this line of code.

```py
from rest_framework import serializers
```
```
Diagnostics:
1. Import "rest_framework" could not be resolved [reportMissingImports]
```

Hopefully someone can make sense of this and help me out :) 

Cheers!

**Comment by intgr** [comment]
> If I'm understanding the docs right. then I am supposed to install the 1.7.0 version to match the mypy version I have?

That's not correct, I think the documentation is unclear. Is there something we can improve in the docs?

The latest version that is strictly compatible with mypy 1.7.0 is  `djangorestframework-stubs==3.14.5` ([setup.py](https://github.com/typeddjango/djangorestframework-stubs/blob/3.14.5/setup.py#L32))

(Note: There's a tradeoff between using older -stubs versions, or updating to latest even if your Django version is not latest. See here for details: https://github.com/typeddjango/django-stubs/discussions/2101#discussioncomment-9276632)

> When I open my editor NeoVim with the virtual environment active I get this error message for this line of code.
> `1. Import "rest_framework" could not be resolved [reportMissingImports]`

This error comes from pylance/pyright, not mypy.

I'm guessing you installed `djangorestframework-stubs` inside a virtualenv, but when NeoVim invokes pyright, it runs outside of that virtualenv and does not find the stubs package. You can probably configure it to use the venv. But sorry, can't help more there, I have not used pyright/NeoVim myself.

**Comment by DMunkei** [comment]
I'm not really experience with mypy and stubs. What would you recommend me doing?  Would appreciate that if it's not too much of an inconveniece. Do I simply just execute these commands?
```
pip install myp
pip install djangorestframework-stubs
pip install django-stubs
```
